src/main/scala/mesosphere/marathon/core/flow/FlowModule.scala

```diff
@@ -0,0 +1,33 @@
+package mesosphere.marathon.core.flow
+
+import mesosphere.marathon.MarathonSchedulerDriverHolder
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.flow.impl.{ OfferMatcherLaunchTokensActor, ReviveOffersActor }
+import mesosphere.marathon.core.leadership.LeadershipModule
+import mesosphere.marathon.core.matcher.manager.OfferMatcherManager
+import mesosphere.marathon.core.task.bus.TaskStatusObservables
+import rx.lang.scala.Observable
+
+class FlowModule(leadershipModule: LeadershipModule) {
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.